src/cf-layout.less

```diff
@@ -1404,6 +1404,281 @@
+            background-size: 350px;
+          }
+          .customized-hero .hero_wrapper {
+              padding-bottom: 225px;
+          }
+          @media only all and (min-width: 37.5em) {
+           .hero .hero_wrapper {
+              padding-right: 200px;
+              padding-bottom: 30px;
+            }
+            .hero .hero_wrapper:after {
+              background-size: 150px;
+              width: 195px;
+            }
+          }
+        </style>
+        <section class="content_hero hero hero__small-content">
+            <div class="hero_wrapper wrapper wrapper__match-content">
+                <div class="hero_text">
+                    <h1 class="hero_heading">Hero title</h1>
+                    <p class="hero_subhead">
+                        Hero text goes here
+                    </p>
+                </div>
+            </div>
+        </section>
+      notes:
+        - "If the hero text is short or doesn't include a call to action, adding
+           the small-content class will increase the top padding value so the
+           text is more vertically aligned with the image."
+      codenotes:
+        - .hero__small-content
+  tags:
+    - cf-layout
+*/
+
+// 300px - 60px for top/bottom padding
+@hero-desktop-height: 240px;
+@hero-image-height: 195px;
+
+.hero {
+  background-color: @gray-10;
+
+  &_wrapper {
+    padding: unit(@grid_gutter-width / @base-font-size-px, em)
+             unit(15px / @base-font-size-px, em);
+
+    .respond-to-min(@tablet-min, {
+      display: table;
+      min-height: unit(@hero-desktop-height / @base-font-size-px, em);
+    });
+
+    .respond-to-min(@desktop-min, {
+      padding: unit(45px / @base-font-size-px, em);
+    });
+
+
+    .lt-ie9 & {
+      // Reduce min-height because IE8 apparently isn't honoring the border-box property,
+      // perhaps due to bug seen here: http://grab.by/JutE
+      min-height: initial;
+    }
+  }
+
+  &_text {
+    .respond-to-min(@tablet-min, {
+      .grid_column(7,12);
+      display: table-cell;
+      vertical-align: middle;
+    });
+  }
+
+  &_heading {
+    .heading-1();
+
+    .respond-to-max(800px, {
+      .heading-2();
+    });
+  }
+
+  &_subhead {
+    .heading-3();
+
+    margin-bottom: unit(@grid_gutter-width / 22px, em);
+
+    .respond-to-max(800px, {
+      .heading-4();
+    });
+
+    &:last-child {
+      margin-bottom: initial;
+    }
+  }
+
+  &_cta {
+    .webfont-regular();
+
+    margin-bottom: initial;
+  }
+
+  &_image {
+    position: relative;
+    min-height: unit(@hero-image-height / @base-font-size-px, em);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+
+    .respond-to-min(@tablet-min,, {
+      .grid_column(5,12);
+      display: table-cell;
+      vertical-align: middle;
+    });
+
+    .respond-to-max(@mobile-max, {
+      margin-top: unit(@grid_gutter-width / @base-font-size-px, em);
+    });
+  }
+}
+
+
+/**
```

Comment:
Shoot, you can remove this once you update to latest `cf-layout` in gh-pages because it _should_ grab the latest release of `cf-core`